test/unit/events.spec.ts

```diff
@@ -504,18 +467,17 @@ describe("Events", () => {
       const flicking = flickingInfo.instance;
 
       flicking.moveTo(2, 200);
-      await waitFor(100);
+      tick(300);
       await simulate(flickingInfo.element, { deltaX: 100 });
-      await waitFor(1000);
 
       expect(flicking.getIndex()).equals(1);
     });
 
     it("will trigger events in correct order even if interrupted", async () => {
-      await simulate(flickingInfo.element, { deltaX: -100, duration: 50 });
-      await waitFor(25);
-      await simulate(flickingInfo.element, { deltaX: -100, duration: 100 });
-      await waitFor(1000);
+      // Don't let first simulate finished, as this is testing interruptablility
+      // Setting time same to duration(400) makes only panning behavior finished, but not animation
+      simulate(flickingInfo.element, { deltaX: -500, duration: 400 }, 400);
+      await simulate(flickingInfo.element, { deltaX: -500, duration: 100 });
```

Comment:
is `await` needed?

Reply:
All `await `is not necessary now.
It isn't important in all of the current tests, but I just left that because in this test we're testing whether interrupting while animating is permitted.
My purpose was to differentiate them. The first one should not be finished, while the second one must be finished.
It has no meaning in the result as you said.

Reply:
@WoodNeck I understand your intent. And I think it's good intent. But what I am concerned is whether we can keep this intention in the future.